sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_hub_operation.py

```diff
@@ -173,6 +175,9 @@ def begin_delete(self, name: str, *, delete_dependent_resources: bool, **kwargs:
             i.e., container registry, storage account, key vault.
             The default is False. Set to True to delete these resources.
         :type delete_dependent_resources: bool
+        :keyword permanently_delete: Workspaces are soft-deleted state by default to allow recovery of workspace data.
```

Comment:
Nit keyword -> param
```suggestion
        :parampermanently_delete: Workspaces are soft-deleted state by default to allow recovery of workspace data.
```